sktime/forecasting/compose/_multiplexer.py

```diff
@@ -6,12 +6,13 @@
 from sktime.forecasting.base._base import DEFAULT_ALPHA
 from sklearn.base import clone
 
-__author__ = ["Kutay Koralturk"]
+__author__ = ["Kutay Koralturk", "Martin Walter"]
 __all__ = ["MultiplexForecaster"]
 
 
 class MultiplexForecaster(_HeterogenousEnsembleForecaster):
```

Comment:
Can you fix the Example section header in the class doc? It should be Examples with correct number of "_" underneath.

Reply:
sure, I am wondering why `pydocstyle` did not catch it